module/ColorJudge.cpp

```diff
@@ -0,0 +1,81 @@
+/**
+ * @file ColorJudge.cpp
+ * @brief 色識別クラス
+ * @author hiroto0927, miyashita64
+ */
+
+#include "ColorJudge.h"
+
+ColorJudge::ColorJudge() {}
+
+// RGBから色を判別する
+COLOR ColorJudge::getColor(rgb_raw_t const& _rgb)
+{
+  rgb_raw_t rgb;
+  hsv_raw_t hsv;
+  int min;
+
+  // 環境光による値の偏りを軽減する
+  // 事前に測った白が(255,255,255)となるように、RGB値を変換する
+  rgb.r = (_rgb.r < MAX_RGB.r) ? _rgb.r * 255 / MAX_RGB.r : 255;
+  rgb.g = (_rgb.g < MAX_RGB.g) ? _rgb.g * 255 / MAX_RGB.g : 255;
+  rgb.b = (_rgb.b < MAX_RGB.b) ? _rgb.b * 255 / MAX_RGB.b : 255;
+
+  // HSV値に変換
+  hsv = ColorJudge::getHsv(rgb);
```

Comment:
```suggestion
  hsv = getHsv(rgb);
```
この記述って同じスコープ内だし冗長じゃない？
つけるとしたら`this->`だろうけど何もなくていいと思うよ